package.json

```diff
@@ -81,19 +81,23 @@
     "core:dev:build": "build-storybook -s ./src/clr-core/.storybook/public -c src/clr-core/.storybook -o dist/storybook/core",
     "core:lint": "npm run prelint && tslint -p . -c ./src/clr-core/tslint.json \"src/clr-core/**/*.ts\" -e \"src/clr-core/**/*.css.ts\" && stylelint --config ./src/clr-core/.stylelintrc \"src/clr-core/**/*.scss\"",
     "core:lint:fix": "npm run prelint && tslint --fix -p . -c ./src/clr-core/tslint.json \"src/clr-core/**/*.ts\" stylelint --config ./src/clr-core/.stylelintrc --fix \"src/clr-core/**/*.scss\"",
+    "core:golden": "npm run core:build:api",
+    "core:golden:fix": "npm run core:golden && cpy dist/clr-core/custom-elements.json ./src/clr-core/",
+    "core:golden:test": "diff -u src/clr-core/custom-elements.json dist/clr-core/custom-elements.json",
```

Comment:
This sets up to use the API metadata for a golden file. Right now it is not source controlled until a PR in the web component analyzer lands.